frontend/src/js/external-forms/form-concept-group/FormConceptCopyModal.tsx

```diff
@@ -38,25 +38,32 @@ const SxInputCheckbox = styled(InputCheckbox)`
   margin: 5px 0;
 `;
 
-interface PropsT {
-  targetFieldname: string;
-  onAccept: (selectedNodes: FormConceptGroupT[]) => void;
-  onClose: () => void;
-}
-
 const FormConceptCopyModal = ({
   targetFieldname,
   onAccept,
   onClose,
-}: PropsT) => {
+}: {
+  targetFieldname: string;
+  onAccept: (selectedNodes: FormConceptGroupT[]) => void;
+  onClose: () => void;
+}) => {
   const { t } = useTranslation();
   const activeLang = useActiveLang();
   const { getValues } = useFormContext();
   const formValues = getValues(); // Isn't watching for changes
   const visibleConceptListFields = useVisibleConceptListFields();
 
   const conceptListFieldOptions = visibleConceptListFields
-    .filter((field) => field.name !== targetFieldname)
+    .filter((field) => {
+      const isAnotherField = field.name !== targetFieldname;
+      const hasValues =
+        formValues[field.name] &&
+        formValues[field.name].some((value: FormConceptGroupT) =>
+          value.concepts.some(exists),
```

Comment:
mega nitpick: but i would probably write this as `formValues[field.name].flatMap(v -> v.concepts).some(exists)`  since you're not interested in the nested context

Reply:
good point, done